README.md

```diff
@@ -1,25 +1,54 @@
 # Webstone
 
-Start your next web application with Webstone and configure it as you go.
+Webstone provides a CLI and an ecosystem of plugins to develop your full-stack web application.
+
+Use the CLI to add REST API endpoints and/or pages. Need a payment integration? Install a payment plugin and use the CLI to create the necessary files and code.
+
+When a plugin author fixes a bug or releases a new feature, you upgrade your plugin dependency and voilà, your project can take advantage of the latest plugin release.
+
+## Getting started
+
+```bash
+npm create webstone-app my-project
+```
+
+or
+
+```bash
+yarn create webstone-app my-project
+```
+
+or
 
 ```bash
 pnpm create webstone-app my-project
 ```
 
-> **Note**: Webstone requires the use of `pnpm` (https://pnpm.io). To install it, please run: `npm i -g pnpm`
+This creates a skeleton [SvelteKit](https://kit.svelte.dev) app and installs the `@webstone/cli` Webstone CLI dev dependency.
+
+> **Note**
+> You can also run the above command in an existing SvelteKit project!
+>
+> This allows anyone to benefit from the Webstone plugin ecosystem, even if you already have an app.
+
+## Plugin ecosystem
 
-What you get with the above command is a full-stack application project - everything you need when you develop a modern web application. More specifically, the project contains:
+Webstone at its core only consists of the `@webstone/cli` CLI. Any Webstone plugin is either created by the Webstone team and hosted in our monorepo or developed by anyone
+from the community.
 
-- Frontend
-  - ✅ [Svelte](https://svelte.dev)
-  - ✅ [SvelteKit](https://kit.svelte.dev)
-  - ⏳️ [GraphQL](https://graphql.org)
-- Backend
-  - ✅ [SvelteKit endpoints](https://kit.svelte.dev/docs#routing-endpoints)
-  - ⏳️ [Prisma](https://www.prisma.io)
-  - ⏳️ Your database of choice
-- Deployment
-  - ⏳️ Your platform of choice
+No matter who authors a plugin, each plugin is created with the following command:`
+
+```
+[npm|yarn|pnpm] create webstone-app my-plugin --type=plugin
```

Comment:
In case no `--type` argument gets provided, you will get asked if you want type application or plugin